src/components/Searchbar/ClearInput.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
-const CloseButton = ({ clearInput }) => {
+const ClearInput = ({ clearInput }) => {
   return (
     <button className="searchbar__button" onClick={clearInput}>
       x
```

Comment:
Could potentially be replaced by an icon, which would look better.

Reply:
dobra uwaga, ale nie chciałem za bardzo tego rozbudowywać, czego chcemy używać?

Reply:
True, we can decide on it later :D